api/PROFILE.md

```diff
@@ -80,6 +80,25 @@ without any ongoing or pending operations left behind.
 When device fails to apply the configuration, it SHOULD eventually stop retrying and publish the new radio status afterwards,
 indicating the error condition inside the `RadioStatus.config_error` field.
 
+### Info
+
+Publish the current state of device to the local server.
+
+POST /api/v1/info
+
+Return codes:
+
+* Success: `200`
+* Not implemented: `404`
+
+Request:
+
+The request mime type MUST be "application/x-proto-binary".
+The request MUST have the body of a single protobuf message of type [DeviceInfo](./proto/profile/local_profile.proto).
```

Comment:
Naming this DeviceInfo is a bit odd if it is just containing App information. Maybe "LocalAppInfo"?

Reply:
I think in future we can expand it to use other info (i.e. info about interfaces or about base image we use, or about available storage). Do you think it is reasonable to prepare another endpoints in the future and use only /api/v1/appinfo for now?

Reply:
Since we don't want the local profile server to grow without bounds I think it makes sense to have very targeted APIs, as we already have for the profile and the radio silence. Hence let's not make it seemingly easier to add more stuff by naming things this way.